app/controllers/internal/users_controller.rb

```diff
@@ -81,8 +81,10 @@ def toggle_ban_user
   def toggle_trust_user
     if user_params[:trusted_user] == "1"
       @user.add_role :trusted
+      @user.trusted
```

Comment:
I believe you should also clear/update cache upon trusting a user. If for some reason `#trusted` was called on user in the past 200 hours (ie async controller), the cache would exist and this line would return false despite the role just got added.

Reply:
Really good point - just updated.